src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx

```diff
@@ -67,6 +67,31 @@ There are several **types** of synthetic monitor to add. **Ping** monitors ensur
 
     Wait a few minutes, then check your monitor from the [**Monitors** index](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-monitors-dashboard-access-your-monitors).
   </Collapser>
+
+   <Collapser
+    id="stepmonitor"
+    title="Add a step monitor"
+  >
+    1. Go to [**one.newrelic.com**](http://one.newrelic.com/) **> Synthetics > <Icon name="fe-plus-circle"/>
+       Create monitor**.
+    2. Select a Step monitor as the [monitor type](#setting-type).
+    3. Specify a [name](#setting-name) and choose a [frequency](#setting-frequency) to determine how often each location will run your monitor.
+    4. Select the [locations](#setting-location) from which you want your monitor to run.
+    5. Build your monitor by selecting from the preconfigured steps at the bottom of the UI:
+        * Navigate to a URL 
+        * Type text
+        * Click an element
+        * Assert text
+        * Assert an element 
+        * Secure a credential
+
+        Use the instructions on the right side of the UI to help locate elements by CSS class, HTML ID, link text, or XPath. 
+    6. Select **Save monitor** to confirm.
+
+  Wait a few minutes, then check your monitor from the [**Monitors** index](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-monitors-dashboard-access-your-monitors).
```

Comment:
Is path and "index" still accurate in NR One UI?

Reply:
Yup! There are some large UI changes upcoming, but those are not rolled out yet.